short_names.json

```diff
@@ -0,0 +1,29 @@
+[{
+    "shortname": "TNN",
+    "fullname": "True-Name Nemesis"
+  }, {
+    "shortname": "mom",
+    "follname": "Mother of Runes"
+  }, {
+    "shortname": "stepmom",
+    "fullname": "Giver of Runes"
+  },
+  {
+    "shortname": "steve",
+    "fullname": "Sakura-Tribe Elder"
+  }, {
+    "shortname": "bob",
+    "fullname": "Dark Confidant"
+  }, {
+    "shortname": "baby jace",
+    "fullname": "Jace, Vryn's Prodigy"
+  },
+  {
+    "shortname": "jtms",
+    "fullname": "Jace, the Mind Sculptor"
+  },

```

Comment:
Inconsistent formatting in the file here "}, {" vs "}, \n {"

Reply:
Fixed. Former over latter, removed errant newlines.